specification/containerservices/resource-manager/readme.md

```diff
@@ -0,0 +1,188 @@
+
+These settings apply only when `--tag=package-2017-09` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-09'
+input-file:
+- Microsoft.ContainerService/2017-07-01/containerService.json
+- Microsoft.ContainerService/2017-08-31/managedClusters.json
+- Microsoft.ContainerService/2017-09-30/location.json
+```
+
+### Tag: package-2017-08
+
+These settings apply only when `--tag=package-2017-08` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-08'
+input-file:
+- Microsoft.ContainerService/2017-07-01/containerService.json
+- Microsoft.ContainerService/2017-08-31/managedClusters.json
+```
+
+### Tag: package-2017-07
+
+These settings apply only when `--tag=package-2017-07` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-07'
+input-file:
+- Microsoft.ContainerService/2017-07-01/containerService.json
+```
+
+### Tag: package-2017-03
+
+These settings apply only when `--tag=package-2017-03` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-03'
+input-file:
+- Microsoft.ContainerService/2017-01-31/containerService.json
+```
+
+### Tag: package-2017-01
+
+These settings apply only when `--tag=package-2017-01` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-01'
+input-file:
+- Microsoft.ContainerService/2017-01-31/containerService.json
+```
+
+### Tag: package-2016-09
+
+These settings apply only when `--tag=package-2016-09` is specified on the command line.
+
+``` yaml $(tag) == 'package-2016-09'
+input-file:
+- Microsoft.ContainerService/2016-09-30/containerService.json
+```
+
+### Tag: package-2016-03
+
+These settings apply only when `--tag=package-2016-03` is specified on the command line.
+
+``` yaml $(tag) == 'package-2016-03'
+input-file:
+- Microsoft.ContainerService/2016-03-30/containerService.json
+```
+
+### Tag: package-2015-11-preview
+
+These setings apply only when `--tag=package-2015-11-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2015-11-preview'
+input-file:
+- Microsoft.ContainerService/2015-11-01-preview/containerService.json
+```
+
+### Tag: package-2015-06-preview
+
+These settings apply only when `--tag=package-2015-06-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2015-06-preview'
+input-file:
+- Microsoft.ContainerService/2015-11-01-preview/containerService.json
+```
+
+---
+
+# Code Generation
+
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  clear-output-folder: true
+```
+
+### Tag: package-2017-01 and go
+
+These settings apply only when `--tag=package-2017-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2017-01' && $(go)
+namespace: containerservice
+output-folder: $(go-sdk-folder)/services/compute/mgmt/2017-01-31/containerservice
```

Comment:
Is the intention here to package the bits along with the compute ones.

Reply:
The containerServices API was previously part of the compute package, but this PR separates it. If someone needs to regenerate the Go SDK with `autorest --tag=package-2017-01` this should produce the same results as before, which seemed to me like the safe approach.

But if you'd rather have the paths changed to say `containerservice` or have these sections removed, let me know.

Reply:
Ahh I see, so I am guessing new packages will be generated at a different path, that sounds okay